core/rawdb/accessors_chain.go

```diff
@@ -545,7 +545,7 @@ func DeleteTermini(db ethdb.KeyValueWriter, hash common.Hash) {
 }
 
 // ReadWorkObjectHeader retreive's the work object header stored in hash.
-func ReadWorkObjectHeader(db ethdb.Reader, hash common.Hash, woType int) *types.WorkObjectHeader {
+func ReadWorkObjectHeader(db ethdb.Reader, hash common.Hash, woType types.WorkObjectView) *types.WorkObjectHeader {
```

Comment:
we shouldnt change this like this, if its a constant, its not an argument

Reply:
The commit this comment is on was refactoring the int to a WorkObjectView type. It was found to not be needed in this case, so was removed as the above comment shows.